### ISSUE TITLE
Wrong version number

### ISSUE THREAD

**Issue by slopp** [rstudio/rsconnect-jupyter#220]
If you run:

```
import rsconnect_jupyter
print(reconnect_jupyter.__version__)
# 1.0.0
```

The result, `1.0.0` is not correct. This same `1.0.0` version number appears when you enable the notebook extension:

```
Enabling: rsconnect_jupyter
- Writing config: /Users/sean/Documents/Sandbox/test-rsconnect-jupyter/env/etc/jupyter
    - Validating...
      rsconnect_jupyter 1.0.0 OK
```

**Comment by slopp** [comment]
This was supposedly fixed in #208 but I am still seeing this with 1.2.2-7 using Python 2.7 in a virtual environment.

**Comment by slopp** [comment]
Things are definitely out of whack with the 1.2.2 release
<img width="869" alt="Screen Shot 2019-12-31 at 10 18 37 AM" src="https://user-images.githubusercontent.com/15901072/71628797-2fd94880-2bb7-11ea-8eeb-da7fcd44440c.png">
<img width="1249" alt="Screen Shot 2019-12-31 at 10 17 04 AM" src="https://user-images.githubusercontent.com/15901072/71628798-2fd94880-2bb7-11ea-871b-f4ec54a50135.png">
<img width="1120" alt="Screen Shot 2019-12-31 at 10 16 10 AM" src="https://user-images.githubusercontent.com/15901072/71628799-3071df00-2bb7-11ea-83e0-c78874d0a49a.png">
<img width="1215" alt="Screen Shot 2019-12-31 at 10 15 47 AM" src="https://user-images.githubusercontent.com/15901072/71628800-3071df00-2bb7-11ea-9a96-cb4577649db4.png">

**Comment by mmarchetti** [comment]
Verified this is fixed - tested rsconnect-python 1.5.0 and rsconnect-jupyter 1.4.0.

```
$ jupyter-serverextension enable --sys-prefix --py rsconnect_jupyter
Enabling: rsconnect_jupyter
- Writing config: /Users/mmarchetti/jtmp/etc/jupyter
    - Validating...
      rsconnect_jupyter 1.4.0 OK
```

```
$ python
Python 3.8.6 (default, Oct  8 2020, 14:07:53)
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rsconnect_jupyter
>>> print(rsconnect_jupyter.__version__)
1.4.0
```
